src/components/LemmaInput.vue

```diff
@@ -0,0 +1,43 @@
+<script setup lang="ts">
+import { TextInput } from '@wmde/wikit-vue-components';
+import { useMessages } from '@/plugins/MessagesPlugin/Messages';
+
+interface Props {
+	modelValue: string;
+	error?: 'ERROR_LEMMA_TOO_LONG' | 'ERROR_NO_LEMMA' | null;
```

Comment:
I like this error key approach, we can definitely develop this further from here. Thank you!

Reply:
Now looking at actually implementing errors, I'm no longer sure this was the best idea^^
At the very least it was too early to do it in this particular pull request